### ISSUE TITLE
Save functionality not working

### ISSUE THREAD

**Issue by outdoortejan** [phcode-dev/phoenix#1710]
### Describe the bug

After saving a file with File > Save File or with the keyboard shortcut  I'll try to close out the Phoenix code application and I am presented with a pop up saying my file is not saved. I select "save file" and try to close the application but the same pop up is shown. If I force close and reopen my changes to the file are gone.

### Expected behavior

Changes since last save are saved after closing and opening file.

### OS, Browser and Phoenix versions

OS: MacOS

**Comment by outdoortejan** [comment]
Yes, the file is a network location. Google drive.

**Comment by abose** [comment]
This is a known issue with some network drives. How is Google drive mounted on your mac, is it any app from Google. Can you give some instructions on how to set up Google drive like your setup for me to recreate this issue.

**Comment by outdoortejan** [comment]
It's mounted via the google drive app for mac:
https://www.google.com/drive/download/

Shouldn't Phoenix Code show some kind of error if its having trouble saving?

**Comment by abose** [comment]
Got it. Will try it out.

**Comment by devishraq** [comment]
Is this an incompatibility between Phoenix and network drives? Specifically, does Phoenix have difficulty interacting with virtual file systems like Google Drive when it's mounted as a network disk?

**Comment by abose** [comment]
Yes, there are known compatibility issues with network drives. This fix is a bit hard as Phoenix keeps track of modified times of files to detect external changes and network drives like gdrive change file modified times unpredictably depending on network sync latency. Since we r getting a few reports, i guess will have to address this soon.
